AirCasting/Settings/MyAccountView.swift

```diff
@@ -0,0 +1,58 @@
+// Created by Lunar on 17/06/2021.
+//
+
+import SwiftUI
+
+struct MyAccountView: View {
+    var body: some View {
+        ZStack {
+            VStack(alignment: .leading) {
+                #warning("Logic here needs to be finished - handle log in")
+                Text("You aren’t currently logged in")
+                    .foregroundColor(.aircastingGray)
+                    .padding()
+                
+                Button(action: {
+                }) {
+                    Group {
+                        HStack {
+                            Text("Create an account")
+                            Spacer()
+                            Image(systemName: "chevron.right")
+                        }
+                        .padding(.horizontal)
+                    }
+                } .buttonStyle(BlueButtonStyle())
+                .padding()
+                
+                Button(action: {
+                }) {
+                    Group {
+                        HStack {
+                            Text("Login in")
+                            Spacer()
+                            Image(systemName: "chevron.right")
+                        }
+                        .padding(.horizontal)
+                        .padding(.vertical, 12)
+                    }
+                    .overlay(
+                        RoundedRectangle(cornerRadius: 5)
+                            .stroke(lineWidth: 1.0)
+                    )
+                }
+                .padding()
+                Spacer()
+            }
+        }
+        .navigationTitle("My account")
+    }
+}
```

Comment:
I think this body would be much more readable if we got rid of the unneeded details at the `body` level. [Top-level functions](https://stackoverflow.com/questions/633617/whats-the-difference-between-low-level-functions-top-level-functions) (computed var here is really a function with no paramaters, `var` is just Swift syntactic sugar) should really be screaming their intent, hiding any unneeded implementation detail in some private functions below. Anyone that wants to know the details would just scroll down to the part they are interested in and read it and everyone elses time is saved by making this high-level concept really easy to read thru. An example how this particular code can be divided:

```Swift
var body: some View {
    ZStack {
        VStack(alignment: .leading) {
            #warning("Logic here needs to be finished - handle log in")
            loggedOutInformationLabel
            createAccountButton
            logInButton
            Spacer()
        }
    }
    .navigationTitle("My account")
}

private var loggedOutInformationLabel: some View {
    Text("You aren’t currently logged in")
        .foregroundColor(.aircastingGray)
        .padding()
}

private var createAccountButton: some View {
    Button(action: {
    }) {
        Group {
            HStack {
                Text("Create an account")
                Spacer()
                Image(systemName: "chevron.right")
            }
            .padding(.horizontal)
        }
    }
    .buttonStyle(BlueButtonStyle())
    .padding()
}

private var logInButton: some View {
    Button(action: {
    }) {
        Group {
            HStack {
                Text("Login in")
                Spacer()
                Image(systemName: "chevron.right")
            }
            .padding(.horizontal)
            .padding(.vertical, 12)
        }
        .overlay(
            RoundedRectangle(cornerRadius: 5)
                .stroke(lineWidth: 1.0)
        )
    }
    .padding()
}
```

This simple little division makes it super-clear how this view looks like at just first glance.